src/components/Sidebar/components/UserCard/UserCard.tsx

```diff
@@ -1,12 +1,105 @@
-import React from "react";
+import React, { useState } from "react";
+
 import { User } from "types";
+import { Color } from "../../../../variables";
+import styles from "./styles.module.css";
 
 type Props = {
   user: User;
 };
 
+const container = {
+  display: "flex",
+  flexDirection: "column",
+  padding: "15px",
+  margin: "0px",
+  width: "200px",
+};
+const divStyles = {
+  margin: "0px",
+  display: "flex",
+  alignItems: "space-between",
+  backgroundColor: Color.LimedSpruce,
+  padding: "15px",
+  width: "200px",
+};
+
+const nameStyles = {
+  margin: "0px",
+  fontSize: "15px",
+  lineHeight: "18px",
+  fontWeight: 600,
+};
+
+const emailStyles = {
+  fontSize: "13px",
+  lineHeight: "18px",
+  fontWeight: 500,
+  color: "hsla(0,0%,100%,.5)",
+};
+
+const textDiv = {
+  margin: "10px",
+};
+
+const imageStyles = {
+  width: "50px",
+  height: "50%",
+  borderRadius: "3px",
+};
+
+const arrowStyles = {
+  justifySelf: "flex-end",
+  fontSize: "15px",
+};
+
+const expandedList = {
+  textAlign: "left",
+  padding: "15px",
+  width: "200px",
+};
+
+const expandedListItem = {
+  fontSize: "15px",
+  lineHeight: "18px",
+  fontWeight: 500,
+  color: Color.CodGray,
+};
+
 const UserCard = (props: Props) => {
-  return <div>{props.user.firstName}!</div>;
+  let [show, setShow] = useState(false);
+
+  return (
+    <div style={container}>
+      <div style={divStyles}>
+        <img style={imageStyles} src="avatar-image.png" />
+        <div style={textDiv}>
+          <p style={nameStyles}>{props.user.firstName}!</p>
+          <a style={emailStyles} href="#">
+            {props.user.email}
+          </a>
+        </div>
+        <img
+          onClick={() => setShow((show = !show))}
```

Comment:
This you should not do. If you want to invert the value of `show`, do it like this:
```suggestion
          onClick={() => setShow((!show))}
```

when you do this `state = !state`, you mutate state directly. [This needs to be avoided](https://reactjs.org/docs/state-and-lifecycle.html#do-not-modify-state-directly)